data/libs/NameGen.lua

```diff
@@ -56,6 +56,79 @@ NameGen = {
 		return Culture:FullName(isfemale, rand)
 	end,
 
+
+--
+-- Function: Names
+--
+-- Create first and surname strings. Both names can be composed of more
+-- than one name. Ex. 'Maria Luisa' or 'van der Velden'.
+--
+-- > name1, name2 = Namegen.Names(isfemale, rand)
+--
+-- Parameters:
+--
+--   isfemale - whether to generate a male or female name. true for female,
+--              false for male
+--
+--   rand - optional, the <Rand> object to use to generate the name. if
+--          omitted/nil, <Engine.rand> will be used
+--
+-- Return:
+--
+--   name1 - a string containing the first name
+--
+--   name2 - a string containing the surname
+--
+--
+-- Availability:
+--
+--   2025
+--
+-- Status:
+--
+--   experimental
+--
```

Comment:
You don't need to remove or edit anything, but I think robn said - way back - that the "availability" and "experimental" tags are deprecated, as they don't really serve any function, I think, but I'm not willing to swear by it.

Reply:
Then I'll probably remove it. It's unnecessary noise.

Reply:
Fixed!